uitools/common/src/ArcGISAuthenticationController.cpp

```diff
@@ -163,7 +165,17 @@ void ArcGISAuthenticationController::handleNetworkAuthenticationChallenge(Networ
     }
     case NetworkChallengeType::ClientCertificate:
     {
-      Q_UNIMPLEMENTED();
+      const auto sslBackend = QSslSocket::activeBackend();
+      if (QSslSocket::activeBackend() != "openssl")
+      {
+        qWarning() << QString("ClientCertificate authentication is not supported with the current SSL backend (%1). ").arg(sslBackend) +
```

Comment:
Would `QStringLiteral` work here? I tend to avoid them when I have runtime args but I am curious so I wanted to ask.

Reply:
I pushed the change. I also realized that just logging to the console is insufficient and we need to fail the challenge since we cannot handle it in these cases.